limiter.go

```diff
@@ -16,6 +16,14 @@ type LimitCounter interface {
 }
 
 func NewRateLimiter(requestLimit int, windowLength time.Duration, counter LimitCounter, keyFuncs ...KeyFunc) *rateLimiter {
+	return newRateLimiter(requestLimit, windowLength, counter, nil, keyFuncs...)
+}
+
+func NewCustomRateLimiter(requestLimit int, windowLength time.Duration, counter LimitCounter, onLimit OnLimitFunc, keyFuncs ...KeyFunc) *rateLimiter {
```

Comment:
I'm not a fan of "Custom" name, perhaps we put this directly on the "NewRateLimiter" method. It is an api breaking change, but, I'm fine with that.